File: C#2/Homeworks/03-Methods/03EnglishDigit/EnglishDigit.cs
using System;
/*
Write a method that returns the last digit of given integer as an English word. 
Write a program that reads a number and prints the result of the method.

test 42 -> two 

Input
•On the first line you will receive a number

Output
•Print the last digit of the number as an English word
 
 */
class EnglishDigit
{
   static void Main()
   {
       int n = int.Parse(Console.ReadLine());
       int lastDigit = n % 10;
       string word = GetDigitAsEnglishWord(lastDigit);
       Console.WriteLine(word);
   }

   static string GetDigitAsEnglishWord(int digit)
   {
       string englishWord = "";
       switch (digit)
       {
           case 0: englishWord = "zero"; break;
           case 1: englishWord = "one"; break;
           case 2: englishWord = "two"; break;
           case 3: englishWord = "three"; break;
           case 4: englishWord = "four"; break;
           case 5: englishWord = "five"; break;
           case 6: englishWord = "six"; break;
           case 7: englishWord = "seven"; break;
           case 8: englishWord = "eight"; break;
           case 9: englishWord = "nine"; break;
           default: break;
       }
       return englishWord;
   }
}

File: JS/HW/Functions/english-digit.js
//Write a method that returns the last digit of given integer as an English word. 
//Write a program that reads a number and prints the result of the method.'
//42 -> two

function getLastDigit(args){
    let number = +args[0],
        word ="",
        digit = number % 10;

    switch(digit){
        case 0 : word = "zero";break;
        case 1 : word = "one";break;
        case 2 : word = "two";break;
        case 3 : word = "three";break;
        case 4 : word = "four";break;
        case 5 : word = "five";break;
        case 6 : word = "six";break;
        case 7 : word = "seven";break;
        case 8 : word = "eight";break;
        case 9 : word = "nine";break;
        default : word = "not a digit";break; 
    }
    console.log(word);
}

getLastDigit([42]);
getLastDigit([100]);
getLastDigit([1573]);